frontend/src/components/common/Header/Cart/Cart.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FiShoppingBag } from 'react-icons/fi';
+import rootState from '@models/rootState';
+import renderProductPrice from '@assets/helper/products';
+import { addToCart, removeFromCart } from './cartSlice';
+import './Cart.scss';
+
+export interface CartProps {
+  id: number;
+  name: string;
+  price: number;
+  quantity: number;
+  size: string;
+  color: string;
+  imageUrl: string;
+}
+const Cart = () => {
+  const { cartList, total } = useSelector((state: rootState) => state.cart);
+  const formattedTotal = renderProductPrice(total);
+  const dispatch = useDispatch();
+  const handleAdjustQuantity = (item: CartProps, option: string): void => {
+    if (option === 'increase') {
+      dispatch(addToCart(item));
+    } else {
+      dispatch(removeFromCart(item));
+    }
+  };
+  return (
+    <div className="cart">
+      <div className="cart__container">
+        {cartList.length === 0 ? (
+          <div className="cart__empty-container">
+            <div className="cart__empty-image">
+              <FiShoppingBag className="cart__empty-image-icon" />
+            </div>
+            <h2 className="cart__empty-title">Your cart is empty </h2>
+            <p className="cart__empty-message">Biscuit oat cake wafer icing ice cream tiramisu pudding cupcake.</p>
+          </div>
+        ) : (
+          <>
+            <div className="cart__header">
+              <h2>My Cart</h2>
+            </div>
+            <div className="cart__list">
+              {cartList.map((item: CartProps) => {
+                const { name, price, quantity, size, color, imageUrl } = item;
```

Comment:
I think you should create a function outside the component for this render, it makes the code more readable